data/presets/man_made/ski_jump.json

```
@@ -0,0 +1,25 @@
+{
+    "icon": "temaki-ski_jumping",
+    "fields": [],
+    "moreFields": [
+        "building"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "jump",
+        "ski jump",
+        "in-run",
+        "inrun",
+        "ski jump structure",
+        "ski jumping hill structure",
+        "ski jump in-run",
+        "ski jumping hill in-run tower"
+    ],
+    "tags": {
+        "man_made": "ski_jump"
+    },
+    "name": "Ski Jumping Hill Structure / In-Run Tower"
```

maybe this could be slightly shortened to the following?

```suggestion
    "name": "Ski Jumping In-Run/Tower"
```

---

While in most of the cases it is true that man-made part is equal to the in-run tower.
It is also not uncommon for the man-made structure to include more than the in-run itself if the terrain is too flat (e.g.: [Holmenkollen](https://upload.wikimedia.org/wikipedia/commons/b/b0/Holmenkollbakken_under_construction_aerial_2.jpg)).
There are also ski jumps on slopes that happen to be just right (e.g. as seen [here](https://upload.wikimedia.org/wikipedia/commons/c/c0/Buskerud_Vikersund_ski_jumping_hills_2010-03-13_K45_K65_0cr.jpg)) and whether they really qualify as "man-made" (or "tower") is debatable.
If shortening of this (quite long I agree) name is desired, I would rather opt for `Ski Jumping Hill Structure`.